Source/MultipartFormData.swift

```diff
@@ -510,9 +558,23 @@ open class MultipartFormData {
 
     // MARK: - Private - Content Headers
 
-    private func contentHeaders(withName name: String, fileName: String? = nil, mimeType: String? = nil) -> HTTPHeaders {
+    private func contentHeaders(
+        withName name: String,
+        fileName: String? = nil,
+        mimeType: String? = nil,
+        encoding: ContentHeaderEncoding?
+    ) -> HTTPHeaders {
         var disposition = "form-data; name=\"\(name)\""
-        if let fileName = fileName { disposition += "; filename=\"\(fileName)\"" }
+
+        if let fileName = fileName {
+            let encodingPrefix: String
+            if let encoding = encoding {
+                encodingPrefix = "*=\(encoding.rawValue)"
```

Comment:
GIST, optional encoding Info with one of:
*=UTF-8
*=iso-8859-1
default is without only
=

Reply:
it look's like its not the client bit the server, the server falls back to iso-8859-1 if there is only `=`.

Reply:
so the server uses the UTF-8 String bit only with iso-8859-1 char set so there are unknown characters. So so with *=UTF-8 we can tell the server explicitly that we are using *=UTF-8